src/hooks/useDialog/useDialog.ts

```diff
@@ -1,15 +1,15 @@
-import { useCallback, useState } from "react";
```

Comment:
We should re-set the use of `useCallback`, per our discussion.

Reply:
done